### PR TITLE
Added a program to find Union of two sorted arrays

### PR DESCRIPTION
**Pull request by shivanithorve01**
## Related Issue
-This issue is associated with issue #1162 

## Proposed Changes
- Added a program to find Union of two sorted arrays 
- Updated readme 

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
| Screenshot #1      |
| ----------- | 
| Output  | 
| 
image |

### CODE REVIEW

**Comment by shivanithorve01** [comment]
My PRs https://github.com/Lakhankumawat/LearnCPP/pull/81 and https://github.com/Lakhankumawat/LearnCPP/pull/496 is already merged into main. Please consider my PR as a higher levels.

**Comment by shivanithorve01** [comment]
If there is need of any change please let me know. I will do it ASAP.

**Comment by Mrjoy832** [comment]
Resolve conflicts

**Comment by shivanithorve01** [comment]
@Mrjoy832  Please check

**Comment by Mrjoy832** [comment]
image
put ticks on correct boxes

**Comment by Mrjoy832** [comment]
> image put ticks on correct boxes

I have edited...from next time you will do by yourself

**Comment by Mrjoy832** [comment]
LGTM🌟

File: A-Array/README.md [hunk 1]
```diff
@@ -541,161 +540,37 @@ For every k size window store every element if negative in deque from rear end u
...
-![image](https://user-images.githubusercontent.com/100334178/164883125-5e188e0f-85fd-4a3e-b0b4-ebe140dc1c53.png)
-![image](https://user-images.githubusercontent.com/100334178/164883138-4b106bd0-79d0-403d-b662-b6540a3ce0bc.png)
-
-### Algorithm
-
-1.Take input the size of the array and the elements of the array.
-2.Consider a function reverse1 which takes the parameters-the array(say arr) and the size of the array(say n).
-3.Inside the function, a new array (with the array size of the first array, arr) is initialized. The array arr[] is iterated from the first element, and each element of array arr[] is placed in the new array from the back, i.e., the new array is iterated from its last element.
-4.In this way, all the elements of the array arr[] are placed reversely in the new array.
-5.Further, we can iterate through the new array from the beginning and print the elements of the array.
```
Comment (Lakhankumawat) [inline A-Array/README.md:681]: Great Job , You erased all this section while resolving merge conflicts .
Also there are spacing in your file name , Your file name should be CamelCased with no spacing on `-` in between .

File: A-Array/README.md [hunk 2]
```diff
@@ -541,161 +540,37 @@ For every k size window store every element if negative in deque from rear end u
...
 
-![image](https://user-images.githubusercontent.com/100334178/164883049-30b0e1a4-8b39-4fe1-b6e7-41d766cdd572.png)
+## OUTPUT : 
 
-![image](https://user-images.githubusercontent.com/100334178/164883155-ce34b2ef-fea9-4e0b-9838-5d18adf02b05.png)
 
-### Algorithm
-The reverse2 method uses a similar code for the inputting and printing of the array. However, we don’t create a new array like the above method. Instead, we reverse the original array itself. In this method, we swap the elements of the array. The first element is swapped with the last element. The second element is swapped with the last but one element and so on. 
-For instance, consider array [1, 2, 3, …., n-2, n-1, n]. We swap 1 with n, 2 with n-1, 3 with n-2 and further. 
+![image](https://user-images.githubusercontent.com/81226536/163536470-3971f593-6a1a-4be1-9461-b7e4c3981457.png)
```
Comment (Lakhankumawat) [inline A-Array/README.md:576]: We don't need outputs .

File: A-Array/README.md [hunk 3]
```diff
@@ -541,161 +540,37 @@ For every k size window store every element if negative in deque from rear end u
...
 
-### Properties
-Time-complexity-->O(n)
-Space-complexity-->O(n)
 
+## Approach :   
+
+Merge all elements of A and B by taking common element once.
+
+**Algorithm :**
```
Comment (Lakhankumawat) [inline A-Array/README.md:551]: Working image of approach is missing .

You need to follow particular guidelines for your README , please follow
these guidelines [click here](https://github.com/Lakhankumawat/LearnCPP#to-add-programs-using-stl-follow-same-approach-as--algorithms) 
also you can see how your README should look like [see here](https://github.com/Lakhankumawat/LearnCPP/blob/main/S-SortingAlgorithms/README.md) 
for more please go through these [here](https://youtu.be/7UgC3HDETRA)

**Review by Lakhankumawat — CHANGES_REQUESTED** [review]

**Comment by Lakhankumawat** [comment]
Due to too many issues and erased existing README data i am closing this PR , please create a new pr with all the changes requested follow README guidelines  and mention this pr in your new pr

**Pull request closed without merging**